src/view/util/CaseDataInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+interface Props {
+  lastUpdated?: string;
+}
+
+const CaseDataInfo: React.FC<Props> = props => {
+  if (!props.lastUpdated) {
```

Comment:
Why are we bailing out of rendering this component if there's no `lastUpdated` prop? Seems like we might still want to show the explanation of where the cases are coming from. perhaps we should just conditionally render the Last Updated paragraph element rather than not rendering the whole component.